### PR TITLE
[Doc Improvement][Updated Toolkit CLI][3639596]

### PR DESCRIPTION
**Pull request by v-ypalikila**
Updated

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/Teams-Toolkit-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/159c8278033d353b003a73fb43eb8005146d35aa/msteams-platform%2Ftoolkit%2FTeams-Toolkit-CLI.md) | 89 | 76 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/06099bd2-e893-4961-be1f-969be605f13f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/Teams-Toolkit-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7c548faa7b661e311cc29cf9fd2c91f1ab7792d9/msteams-platform%2Ftoolkit%2FTeams-Toolkit-CLI.md) | 89 | 76 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/38c2150a-f86b-49b9-ae4c-3828ad10478f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/Teams-Toolkit-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fb82d68d05f89dbd13b24ff6c73926240ec03992/msteams-platform%2Ftoolkit%2FTeams-Toolkit-CLI.md) | 89 | 72 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b80324cd-9c6c-40ec-ae28-f708996e2ac7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/Teams-Toolkit-CLI.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Teams Toolkit Command Line Interface
 author: MuyangAmigo
-description: In this module, learn Teamsapp Command Line Interface, TeamsFx library, supported commands and its scenarios.
+description: In this module, learn Teamsapp Command Line Interface, TeamsFx library, supported commands, and its scenarios.
 ms.author: zhany
```
Comment (MuyangAmigo) [inline msteams-platform/toolkit/Teams-Toolkit-CLI.md:5]: Can we remove TeamsFx library? This is CLI doc, should not concern TeamsFx anymore, which was our SDK.

File: msteams-platform/toolkit/Teams-Toolkit-CLI.md [hunk 2]
```diff
@@ -12,27 +12,20 @@ zone_pivot_groups: toolkit-cli
 # Teams Toolkit command line interface
 
 > [!IMPORTANT]
->
-> * Teams Toolkit CLI v3 is available in beta version.
-> * We recommend that you use Teams Toolkit CLI v3 to build your Teams app. TeamsFx CLI v1 and TeamsFx CLI v2 are soon deprecated.
+> We recommend that you use Teams Toolkit CLI v3 to build your Teams app. TeamsFx CLI v1 and TeamsFx CLI v2 are soon deprecated.
```
Comment (MuyangAmigo) [inline msteams-platform/toolkit/Teams-Toolkit-CLI.md:16]: Can we mention that v1 is no longer supported, v2 will be end of support by March 2025 (One year after v3 officially released)

**Review by MuyangAmigo — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/Teams-Toolkit-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f49da31f362c47d77a11faac7fefe3d66f442908/msteams-platform%2Ftoolkit%2FTeams-Toolkit-CLI.md) | 89 | 72 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/41e73831-7507-435f-8aa7-ba781848dbd7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/Teams-Toolkit-CLI.md (hunk 1 above)
Comment (v-ypalikila) [reply msteams-platform/toolkit/Teams-Toolkit-CLI.md:5]: removed.

File: msteams-platform/toolkit/Teams-Toolkit-CLI.md (hunk 2 above)
Comment (v-ypalikila) [reply msteams-platform/toolkit/Teams-Toolkit-CLI.md:16]: Added note that TeamsFx CLI v1 isn't supported and TeamsFx CLI v2 will be depreated by March 2025..

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/Teams-Toolkit-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/50166476162a177f684392c31718ff6005b90a2f/msteams-platform%2Ftoolkit%2FTeams-Toolkit-CLI.md) | 89 | 72 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1bedf45e-134c-4e95-8b54-263cca2081ec) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/Teams-Toolkit-CLI.md [hunk 3]
```diff
@@ -200,6 +195,66 @@ Create a new environment by copying from the existing dev environment:
...
+
+Sideload the Outlook add-in application package with XML based manifest to Outlook.
+
+```bash
+teamsapp install --xml-path manifest.xml
+```
+
+## `teamsapp launchinfo`
+
+Get launch information of an acquired Microsoft 365 app.
```
Comment (jayzhang) [inline msteams-platform/toolkit/Teams-Toolkit-CLI.md:233]: an installed Microsoft 365 app?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/Teams-Toolkit-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cf1c7b6a67fe1ef5044eb53894e100f29f183877/msteams-platform%2Ftoolkit%2FTeams-Toolkit-CLI.md) | 89 | 72 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d9a1a026-9a5c-4b8e-aa61-e84e897c517b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/Teams-Toolkit-CLI.md (hunk 3 above)
Comment (v-ypalikila) [reply msteams-platform/toolkit/Teams-Toolkit-CLI.md:233]: Updated.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/Teams-Toolkit-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/97c2040576fc7d64378bf0883bdf015a30279d52/msteams-platform%2Ftoolkit%2FTeams-Toolkit-CLI.md) | 89 | 72 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eeb25151-0233-4ba4-ac4b-14ceb7b18a04) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by MuyangAmigo — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/Teams-Toolkit-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/abb7e0239444e03ab4443091e6e326b8b7b5d1e5/msteams-platform%2Ftoolkit%2FTeams-Toolkit-CLI.md) | 89 | 72 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c6f1cc04-1cb8-47d5-ac96-46374fbcdd5f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Pull request merged**